docs/cluster.md

```diff
@@ -152,8 +151,6 @@ ports
 
 When forming a cluster across multiple machines:
 * you have to take care of [Extra Network Configuration](#extra-network-configuration)
-* you also can't use `JF_DIST_MODE="sname"` as you have to name Jellyfish nodes using their publicly available IP address
-or domain names (see `JF_DIST_NODE_NAME` and `JF_DIST_MODE`)
```

Comment:
I am not sure about this one so maybe it's better to remove this sentence. If shortname is resolvable, everything should work, and it also depends what do we mean by `globally`

Reply:
Yeah, that's right, if the short name is resolvable, everything will work, but this assumes the setup of the host OS that he will make it resolvable or that jellyfishes will be connected to the same docker network. If you want we can remove these lines, but IMO default mode when running jellyfishes on multiple machines should use longnames.